src/client/gemini_client.rs

```diff
@@ -67,45 +66,54 @@ impl GeminiClient {
         &self,
         batch: &Vec<String>,
         dim: Option<usize>,
+        model: String,
     ) -> Result<Vec<Vec<f32>>, ShellError> {
-        let dimension = match dim {
-            Some(d) => {
-                if d > MAX_EMBEDDING_DIMENSION || d < 1 {
+        if !model.contains("models/") {
```

Comment:
Could we just prefix this automatically if missing and log something at info?

Reply:
Done!